<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteDialog</class>
 <widget class="QWidget" name="DeleteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>381</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>381</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>660</width>
    <height>381</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="delete_title">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>39</y>
     <width>99</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(25,25,25,1)</string>
   </property>
   <property name="text">
    <string>删除操作</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>660</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(206, 220, 245, 1);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="delete_content">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>139</y>
     <width>166</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(51, 51, 51, 1)</string>
   </property>
   <property name="text">
    <string>删除2项内容</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="delete_tip">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>205</y>
     <width>566</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(102, 102, 102, 1)</string>
   </property>
   <property name="text">
    <string>删除的瑞记将进入回收站，30天后将自动永久删除。</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_confirm">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>279</y>
     <width>160</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 154, 252, 1);
border-radius:10px;
color:rgba(255, 255, 255, 1)</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="PictureBox" name="btn_delete" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureBox</class>
   <extends>QWidget</extends>
   <header>picturebox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
